common/README.md

```diff
@@ -30,7 +30,7 @@ Either [Trust the Certs](#trust-the-certs) or [Disable SSL Verification](#disabl
 Edit the `howso.yml` file and set `verify_ssl: false.
 
 
-### Trust the Certs 
+### Trust the Certs
 
 The Howso Platform python client uses the _certifi_ package for the trusted root certs, not the operating system trust store.  Therefore, to trust the platform ca - we'll extract it from the platform and then set it explicitly as trusted in our `howso.yml`
```

Comment:
`Python` should probably be capitalized here. Also `CA` since it is an abbreviation. I know you didn't touch these here, but they could easily be improved in this PR.